src/main/java/Main.java

```diff
@@ -0,0 +1,8 @@
+import com.epam.tat.module4.Calculator;
+
+public class Main {
```

Comment:
Why do you need this class?

Reply:
Played with a calculator, as a self-check (deleted)